packages/typescript/test/ts_project/js_library/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+load("@npm_bazel_typescript//:index.bzl", "ts_project")
+load("//internal/js_library:js_library.bzl", "js_library")
+
+js_library(
+    name = "lib_a",
+    srcs = [
+        "a.d.ts",
```

Comment:
any thought about how these would typically be constructed? is this just for the case where you vendor a node_module? otherwise I'm not sure why you'd be authoring .js+.d.ts and if you didn't author it you'd get it through a node_module_lib